dist/systemd/obsapidelayed-clockworkd.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description = OBS API Delayed clockworkd server
```

Comment:
What is delayed about the clockwork daemon? :rofl: 

I would suggest: `Open Build Service Clockwork Daemon` and renaming the unit to `obs-clockwork.service`

Reply:
@hennevogel 

> I would suggest: `Open Build Service Clockwork Daemon` and renaming the unit to `obs-clockwork.service`

Changed.